src/Views/AccountView.vala

```diff
@@ -175,15 +173,15 @@ public class Installer.AccountView : AbstractInstallerView {
 
         if (username_entry_text == "") {
             username_error_revealer.reveal_child = false;
-            username_entry.set_icon_from_icon_name (Gtk.EntryIconPosition.SECONDARY, "dialog-information-symbolic");
+            username_entry.set_icon_from_icon_name (Gtk.EntryIconPosition.SECONDARY, null);
         } else if (username_is_valid && !username_is_taken) {
             username_error_revealer.reveal_child = false;
             username_entry.set_icon_from_icon_name (Gtk.EntryIconPosition.SECONDARY, "process-completed-symbolic");
         } else {
             if (username_is_taken) {
                 username_error_revealer.label = _("Username is already taken");
             } else if (!username_is_valid) {
-                username_error_revealer.label = _("Username is not valid");
+                username_error_revealer.label = _("Username can only contain lowercase letters, numbers, and no spaces");
```

Comment:
I suggest that "Username can only contain lowercase letters and numbers, without spaces" is clearer and beetter grammar.